mathics/format/pane_text.py

```diff
@@ -0,0 +1,465 @@
+"""
```

Comment:
All this module can be removed once `prettyprint.py` be reformulated using `sympy.prettyprint.stringpict.prettyForm`.

Reply:
A step in the right direction!